shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -554,15 +554,17 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
 
     @Override
     public final WindowInsets onApplyWindowInsets(WindowInsets insets) {
-        // On Android, we do not differentiate between 'safe areas' and view insets.
+        // Status bar, left/right system insets are padding.
```

Comment:
Maybe just as a mini hint for readers, mention top left right are paddings because they're partial obstructions, bottom is generally a full obstruction, thus a view inset.

Reply:
Done. Hinted at rationale: padding is for view content that should sit 'under' the system chrome; insets are for view content where system chrome should effectively 'shrink' the content area.